### ISSUE TITLE
Store re-frisk app state in browser local storage for easier debugging over refreshes

### ISSUE THREAD

**Issue by Trpal** [flexsurfer/re-frisk#71]
I at least personally often end up refreshing when debugging stuff to see how stuff play out, so it would be nice to have re-frisk in the same state after refresh.

This is also a feature from reframe-10x and I could maybe whip up a PR if agreed that this would be beneficial.

**Comment by flexsurfer** [comment]
yeah, that would be cool. i thought about that, but have no time to implement it, it should be a simple one
